org.racket_lang.Racket.yml

```diff
@@ -0,0 +1,34 @@
+app-id: org.racket_lang.Racket
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: drracket
+rename-desktop-file: drracket.desktop
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=x11
+  - --filesystem=home
+  - --filesystem=/tmp
+  - --filesystem=/var/tmp
+modules:
+  - name: racket
+    no-autogen: true
+    subdir: src
+    build-options:
+      # by default, racket binaries are already stripped, so no debug info has to be extracted. 
```

Comment:
maybe build without stripping instead?

Reply:
what's the benefit of doing the way you are suggesting?

I personally prefer this setup,  but I am open to changes. (and I am new to Flatpak, too)